app/components/nav-header.tsx

```diff
@@ -24,27 +29,37 @@ export function NavHeader() {
 					className={clsx(
 						"flex text-sm font-mono border rounded-lg divide-x dark:(border-neutral-700 divide-neutral-700) overflow-hidden",
 						{
-							"!border-blue-500/70": isLogs,
-							"!border-rose-500/70": isRequests,
+							"!border-blue-500/70": isLogs && appCount > 0 && appId,
+							"!border-rose-500/70": isRequests && appCount > 0 && appId,
 						},
 					)}
 				>
-					<Link
-						className={clsx("bg-zinc-100 dark:bg-neutral-800 px-2 py-1", {
-							"!bg-rose-500 text-white": isRequests,
-						})}
-						to={`/app/${params.app}/requests`}
-					>
-						Requests
-					</Link>
-					<Link
-						className={clsx("bg-zinc-100 dark:bg-neutral-800 px-2 py-1", {
-							"!bg-blue-500 text-white": isLogs,
-						})}
-						to={`/app/${params.app}/logs`}
-					>
-						App Logs
-					</Link>
+					{appCount > 0 && appId ? (
```

Comment:
Here too

```suggestion
					{appId ? (
```

should be enough

Reply:
correct